inc/ticket.class.php

```diff
@@ -1886,7 +1886,7 @@ function manageValidationAdd($input) {
          if (count($validations_to_send)) {
             $values                = array();
             $values['tickets_id']  = $this->fields['id'];
-            if ($input['id'] != $this->fields['id']) {
+            if (isset($input['id']) && $input['id'] != $this->fields['id']) {
```

Comment:
I looked over to your remark, I'm not sure but it seems more logical to me if it does not exist.